ARCs/arc-0200.md

```diff
@@ -0,0 +1,232 @@
+---
+arc: 200
+title: Algorand Smart Contract Token Specification
+description: Base specification for tokens implemented as smart contracts
+author: Nicholas Shellabarger (@temptemp3)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/223
+status: Draft
+type: Standards Track
+category: Interface
+created: 2023-07-03
+requires: 3, 4, 22, 28
+---
+
+# Algorand Smart Contract Token Specification
+
+## Abstract
+
+This ARC (Algorand Request for Comments) specifies an interface for tokens to be implemented on Algorand as smart contracts. The interface defines a minimal interface required for tokens to be held and transferred, with the potential for further augmentation through additional standard interfaces and custom methods.
+
+## Motivation
+
+Currently, most tokens in the Algorand ecosystem are represented by ASAs (Algorand Standard Assets). However, to provide rich extra functionality, it can be desirable implementing tokens as a smart contract instead. To foster an interoperable token ecosystem, it is necessary that the core interfaces for tokens be standardized.
```

Comment:
```suggestion
Currently, most tokens in the Algorand ecosystem are represented by ASAs (Algorand Standard Assets). However, to provide rich extra functionality, it can be desirable to implement tokens as smart contracts instead. To foster an interoperable token ecosystem, it is necessary that the core interfaces for tokens be standardized.
```